core/src/test/java/org/apache/iceberg/catalog/CatalogTests.java

```diff
@@ -799,6 +799,23 @@ public void testDropTable() {
     Assert.assertFalse("Table should not exist after drop", catalog.tableExists(TABLE));
   }
 
+  @Test
+  public void testDropTableWithPurge() {
+    C catalog = catalog();
+
+    if (requiresNamespaceCreate()) {
+      catalog.createNamespace(NS);
+    }
+
+    Assertions.assertThat(catalog.tableExists(TABLE)).isFalse();
+
+    catalog.buildTable(TABLE, SCHEMA).create();
+    Assertions.assertThat(catalog.tableExists(TABLE)).isTrue();
+
+    Assertions.assertThat(catalog.dropTable(TABLE, true)).isTrue();
+    Assertions.assertThat(catalog.tableExists(TABLE)).isFalse();
```

Comment:
This looks copied from the test above. Can you copy the assertions with context strings?

Reply:
done